usaspending_api/api_contracts/contracts/v2/download/count.md

```diff
@@ -0,0 +1,132 @@
+
+## Filter Objects
+### FilterObject (object)
++ `keywords` : poptarts (optional, array[string])
++ `time_period` (optional, array[TimePeriodObject], fixed-type)
++ `place_of_performance_scope` (optional, enum[string])
+    + domestic
+    + foreign
++ `place_of_performance_locations` (optional, array[LocationObject], fixed-type)
++ `agencies` (optional, array[AgencyObject])
++ `recipient_search_text`: `Hampton` (optional, array[string])
++ `recipient_id` (optional, string)
+    A hash of recipient DUNS, name, and level. A unique identifier for recipients, used for profile page urls.
++ `recipient_scope` (optional, enum[string])
+    + domestic
+    + foreign
++ `recipient_locations` (optional, array[LocationObject])
++ `recipient_type_names`: `category_business` (optional, array[string])
+    See options at https://github.com/fedspendingtransparency/usaspending-api/wiki/Recipient-Business-Types
++ `award_type_codes` (optional, FilterObjectAwardTypes)
+    See use at
+    https://github.com/fedspendingtransparency/usaspending-api/wiki/Search-Filters-v2-Documentation#award-type
++ `award_ids`: SPE30018FLGFZ, SPE30018FLJFN (optional, array[string])
++ `award_amounts` (optional, array[AwardAmounts])
++ `program_numbers`: `10.331` (optional, array[string])
++ `naics_codes`: 311812 (optional, array[string])
++ `psc_codes`: 8940, 8910 (optional, array[string])
++ `contract_pricing_type_codes`: `J` (optional, array[string])
++ `set_aside_type_codes`: `NONE` (optional, array[string])
++ `extent_competed_type_codes`: `A` (optional, array[string])
++ `tas_codes` (optional, array[TASCodeObject], fixed-type)
+
+### TimePeriodObject (object)
++ `start_date`: `2017-10-01` (required, string)
++ `end_date`: `2018-09-30` (required, string)
++ `date_type` (optional, enum[string])
+    + action_date
+    + last_modified_date
+
+### LocationObject (object)
++ country: `USA` (required, string)
++ state: `VA` (optional, string)
++ county (optional, string)
++ city (optional, string)
+
+### AgencyObject (object)
++ type (required, enum[string])
+    + awarding
+    + funding
++ tier (required, enum[string])
+    + toptier
+    + subtier
++ name: `Department of Defense` (required, string)
+
+### AwardAmounts (object)
++ `lower_bound` (optional, number)
++ `upper_bound`: 1000000 (optional, number)
+
+### TASCodeObject (object)
++ `ata` (optional, string, nullable)
+    Allocation Transfer Agency Identifier - three characters
++ `aid` (required, string)
+    Agency Identifier - three characters
++ `bpoa` (optional, string, nullable)
+    Beginning Period of Availability - four digits
++ `epoa` (optional, string, nullable)
+    Ending Period of Availability - four digits
++ `a` (optional, string, nullable)
+    Availability Type Code - X or null
++ `main` (required, string)
+    Main Account Code - four digits
++ `sub` (optional, string, nullable)
+    Sub-Account Code - three digits
+
+### FilterObjectAwardTypes (array)
+List of filterable award types
+
+### Sample
```

Comment:
Are `Sample` and `Default` data structures? Are they being used in this file?

Reply:
I believe they are used as part of the `FilterObjectAwardTypes`. I am going to change the header value level to make it more obvious.